Autodesk/synthesis
# Main HUD Improvements

### Description
Improved the main HUD to fit with the Figma design, including a gradient background, the Autodesk logo, two separated buttons, and two item trays. The color manager sets the colors, so the default color scheme should be updated later. The hover effect in Figma is a lighter color than the background, but I don't think that's possible with the built-in color tint transition for Unity buttons. Luca is working on something that would make this possible, so I will update that once his is merged.

![Screenshot 2023-07-20 at 4 54 06 PM](https://github.com/Autodesk/synthesis/assets/20529380/e1147ceb-e850-4fa9-9be2-2f5786786b5f)

### Objectives
- [X] Updated Main HUD design
- [X] Item hover effect
- [X] Open and close HUD *(tween library breaks sometimes and causes the movement to not happen)*
- [X] Update entries in practice mode
- [X] Update entries in match mode

### Testing Done
- Entered in modes available from main menu
- Clicked buttons to make sure modals/panels appear
- Change colors in theme manager
- Open and close a lot (after restarting Unity so tween library works)

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1575)
